ee/localserver/server.go

```diff
@@ -104,9 +104,23 @@ func New(ctx context.Context, k types.Knapsack, presenceDetector presenceDetecto
 	// /v0/cmd left for transition period
 	mux.Handle("/v1/cmd", ecKryptoMiddleware.Wrap(ls.munemoCheckHandler(ecAuthedMux)))
 
+	dt4aKeys, err := dt4aKeys()
```

Comment:
Ah sorry, should've noticed earlier -- if you end up pushing other changes, could you also rename this variable to `trustedDt4aKeys` or `keys` to avoid shadowing?

Reply:
done

Reply:
almost wonder if we should look for a "shadow linter"

Reply:
There are a couple options! I think I looked at enabling the golangci-lint `govet => shadow` option, but didn't end up turning it on because it's idiomatic to shadow `err` and `ctx`, so we have a bunch of violations for that. But maybe it would be simple to add an exception for `err` and `ctx`.